.github/workflows/notebooks.yml

```diff
@@ -0,0 +1,42 @@
+name: Test notebooks
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 12
+      matrix:
+        os: [Ubuntu-20.04, macOS-latest]
```

Comment:
Any reasons to not add windows, too?

Reply:
Only because I expected it would require special windows-specific handling to make it work... I just tried it and it seems that's the case, see: https://github.com/numpy/numpy-tutorials/runs/7309090446?check_suite_focus=true

My vote is for incremental adoption: get linux/macos in sooner since they're working now, and open a follow-up issue for anyone who wants to figure out what magic combo of workarounds is necessary to get it to work on windows :)